PocketKit/Sources/Sync/PersistentContainer.swift

```diff
@@ -15,7 +15,7 @@ public class PersistentContainer: NSPersistentContainer {
     }()
 
     private (set) var spotlightIndexer: CoreDataSpotlightDelegate?
-    private (set) var storeDescription: NSPersistentStoreDescription?
+    private (set) var storeDescription: NSPersistentStoreDescription!
```

Comment:
what's the advantage of implicitly unwrapping here?

Reply:
`getStoreDescription` returns non-optional.
So I removed the optionalness of `storeDescription` and xcode complained about not all properties being initalised before the class called `super.init`.
`storeDescription` will be valid before it's accessed or something horrifying has happened and so we should probably crash out.